src/app/app-routing.module.ts

```diff
@@ -54,9 +55,10 @@ export const routes: Routes = [
   { path: 'users', component: OtherUserProfilesComponent },
   { path: 'users/:userId/skills', component: OtherUserSkillsComponent },
   { path: 'search-users', component: SearchUsersComponent },
+  { path: 'sales-search', component: SalesSearchComponent },
   { path: 'communities', component: CommunitiesComponent },
   { path: 'communities/:communityId', component: CommunityViewComponent },
-  { path: '', redirectTo: 'my-skills', pathMatch: 'full' },
+  { path: '', redirectTo: 'sales-search', pathMatch: 'full' },
```

Comment:
Although this view will be primarily used by sales and managers we should name it more general because it will be open for any user. For example: "user-profile-search"

Reply:
Thank you. Renamed